src/cli/sql.rs

```diff
@@ -216,34 +216,44 @@ fn process(pretty: bool, json: bool, res: surrealdb::Result<Response>) -> Result
 	// Get the number of statements the query contained
 	let num_statements = response.num_statements();
 	// Prepare a single value from the query response
-	let value = {
-		let mut output = Vec::<Value>::with_capacity(num_statements);
-		for index in 0..num_statements {
-			output.push(match response.take(index) {
-				Ok(v) => v,
-				Err(e) => e.to_string().into(),
-			});
-		}
-		Value::from(output)
-	};
+	let mut output = Vec::<Value>::with_capacity(num_statements);
+	for index in 0..num_statements {
+		output.push(match response.take(index) {
+			Ok(v) => v,
+			Err(e) => e.to_string().into(),
+		});
+	}
+
 	// Check if we should emit JSON and/or prettify
 	Ok(match (json, pretty) {
 		// Don't prettify the SurrealQL response
-		(false, false) => value.to_string(),
+		(false, false) => Value::from(output).to_string(),
 		// Yes prettify the SurrealQL response
-		(false, true) => format!("{value:#}"),
+		(false, true) => output
+			.iter()
+			.enumerate()
+			.map(|(i, v)| format!("-- Query {:?}\n{v:#}", i + 1))
```

Comment:
How about including extra whitespace between statement results like this?

```suggestion
			.map(|(i, v)| format!("-- Query {:?}\n{v:#}\n", i + 1))
```

Reply:
Tobie did not want that whitespace